eng/Build.ps1

```diff
@@ -549,11 +549,7 @@ try {
     }
 
     if ($testBenchmarks) {
-        $properties_storage = $properties
-        $properties += "/p:RuntimeIdentifier=win-x64"
-        $properties += "/p:Configuration=Release" # Always run in release.
         BuildSolution "FSharp.Benchmarks.sln" $False
-        $properties = $properties_storage
```

Comment:
I'm almost certain that it won't build without at least the RID.

Reply:
Nono looks like it builds and works like this.